lib/Errors.ml

```diff
@@ -145,11 +156,9 @@ let fatal_error ?(msg = "") _ =
 (* Warn that a language feature is deprecated *)
 let warn_deprecated (pos, msg) =
   let file, line, column = position pos in
-  Printf.eprintf
-    "\n\
-     Warning: deprecated language construct used at file \"%s\", line %d, \
-     column %d:\n"
-    file line (column - 1) ;
+  Printf.eprintf "\nWarning: deprecated language construct used at file %s:\n"
+    (append_position_to_filename file
```

Comment:
Seeing a lot of these warning calls, maybe abstract to a function that emits a warning, file name + line nums, and text at the location?

Reply:
Do you mean to achieve more code sharing between the different kinds of errors and the warnings? They're all a tiny bit different, but I agree there probably could be more code sharing there.

Reply:
It's related. This is just one specific area where we need to DRY up the code.

Reply:
What does it my to DRY up the code? Is that an acronym?

Reply:
yep - "Don't Repeat Yourself." http://wiki.c2.com/?DontRepeatYourself. People usually moderate that with something like "write everything (twice | thrice | ? ) before abstracting". So it's a balance.

Reply:
Cool! I agree that the error messages could be condensed further. Hope to address it in a separate issue: https://github.com/stan-dev/stanc3/issues/31 .